### ISSUE TITLE
problem implementing your model

### ISSUE THREAD

**Issue by colinmolter** [nshaud/DeepNetsForEO#6]
Hello, I just tried to implement your model. I compiled caffe successfully. i adapted the script to fit my tiles. I get that issue:

> ....
> I0221 05:26:39.852553  1508 net.cpp:408] conv1_1 <- data
> I0221 05:26:39.852568  1508 net.cpp:382] conv1_1 -> conv1_1
> I0221 05:26:39.855118  1508 net.cpp:124] Setting up conv1_1
> I0221 05:26:39.855144  1508 net.cpp:131] Top shape: 10 64 128 128 (10485760)
> I0221 05:26:39.855150  1508 net.cpp:139] Memory required for data: 45875200
> I0221 05:26:39.855170  1508 layer_factory.hpp:77] Creating layer conv1_1_bn
> I0221 05:26:39.856801  1508 net.cpp:86] Creating Layer conv1_1_bn
> I0221 05:26:39.856830  1508 net.cpp:408] conv1_1_bn <- conv1_1
> I0221 05:26:39.856847  1508 net.cpp:369] conv1_1_bn -> conv1_1 (in-place)
> F0221 05:26:39.856976  1508 batch_norm_layer.cpp:44] Check failed: this-> layer_param_.param(i).lr_mult() == 0.f (0.5 vs. 0) Cannot configure batch normalization statistics as layer parameters.

>  *** Check failure stack trace: ***

`

any idea about what to do, where to check?
thx
colin

**Comment by nshaud** [comment]
You're right. Since a recent Caffe update, the `lr_mult` option does not apply to the BachNormalization layer anymore, I fixed the scripts to reflect this.

Please pull the last update and confirm that this works for you.
